k8s/README.md

```diff
@@ -0,0 +1,30 @@
+# NO!
+Experimental, for testing, don't use in production or if you do fix all the things.
+
+Based on https://kubernetes.io/docs/tutorials/stateful-application/cassandra/
+
+this are purely notes
```

Comment:
```suggestion
These are purely notes
```

Reply:
fixed locally with other changes to reduce commit clutter